Style help and system pages

ciar4n opened the pull request:
### Before
![image](https://user-images.githubusercontent.com/2803503/42136912-884af3c6-7d5b-11e8-9c3b-bff62ea4288a.png)


### After
![image](https://user-images.githubusercontent.com/2803503/42136908-783e6238-7d5b-11e8-8e29-15d79ade5849.png)

wilsonge commented:
![screen shot 2018-07-01 at 21 54 54](https://user-images.githubusercontent.com/1986000/42138693-6ac6be48-7d79-11e8-96fe-083a73e00248.png)

We have a php notice here - this is because in the definitions the icons are associated with the items not the headers. Needs a small tweak to get that working.

We need to think about how this looks on screens too - because even on my large pc screen (30 odd inches) - this is going off the edge of my screen. On my macbook screen you can only see the outer edge of the 4th column

![screen shot 2018-07-01 at 21 55 57](https://user-images.githubusercontent.com/1986000/42138698-8f9c53a4-7d79-11e8-8d4e-f6e2039f006c.png)

Still a big improvement tho! Looking good!

brianteeman commented on administrator/components/com_cpanel/tmpl/help/default.php:
```diff
@@ -11,17 +11,14 @@
-		</div>
-	<?php endforeach; ?>
+<div class="com-cpanel-help">
+	<h4 class="com-cpanel-help__header"><span class="fa fa-<?php echo $link['icon']; ?>"></span><?php echo Text::_('MOD_MENU_HELP'); ?></h4>
```
please add aria-hidden here and to other places in this PR with icons

brianteeman commented:
4 columns is probably a bit adventurous

dgrammatiko commented:
@brianteeman that was the design given

@ciar4n @wilsonge about the notification: I guess I included icons for each menu but not for the headers :(

brianteeman commented:
@wilsonge i think its going off your screen just because of the error as without the error i cant replicate

Currently this isnt completely matching the issue #430 as the view is the same at all screen sizes

wilsonge commented:
Yup - but in the designs none of the actual items have icons (aside from messages). So I guess it's best to do have arrays with

`name`, `icon`, `links`

ciar4n commented:
For now I have removed the icons. Maybe someone with a little more smarts than I, could add them back in on another pr.

4 columns should be fine on most screen sizes. I suspect the overflow issue is related to the long strings in the php error. I added some media queries for the column count on smaller screens.
